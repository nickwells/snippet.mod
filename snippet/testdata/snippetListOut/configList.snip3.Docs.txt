
        Note: snip3 - Note
