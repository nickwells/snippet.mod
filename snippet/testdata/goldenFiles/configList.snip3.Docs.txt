in: testdata/testListConfig

        Note: snip3 - Note
